### PR TITLE
Added Django 2.2 to testing matrix

### PR DESCRIPTION
**Pull request by rossmechanic**
Also fixed a doc issue. Doc was formatted as md instead of rst

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ env:
   - DJANGO="Django>=1.11,<1.12"
   - DJANGO="Django>=2.0,<2.1"
   - DJANGO="Django>=2.1,<2.2"
+  - DJANGO="Django>=2.2,<2.2"
```
Comment (ThePumpingLemma) [inline .travis.yml]: `<2.3`?

Comment (rossmechanic) [reply .travis.yml]: Ah thank you. Actually just tried running the tests locally and they failed 😞

File: runtests.py [hunk 2]
```diff
@@ -43,6 +52,7 @@
             'ENGINE': 'django.db.backends.sqlite3',
         }
     },
+    MIGRATION_MODULES=DisableMigrations(),
```
Comment (kseever) [inline runtests.py]: Might be a dumb question, but why not just `{}`?

File: simple_history/tests/tests/test_models.py [hunk 3]
```diff
@@ -1441,6 +1445,9 @@ def test_foreign_key_to_self_using_self_str(self):
 
 @override_settings(**database_router_override_settings)
 class MultiDBExplicitHistoryUserIDTest(TestCase):
+    if django.VERSION >= (2, 2):
```
Comment (kseever) [inline simple_history/tests/tests/test_models.py]: Why is this version spec different from the other two in this file? And if it's not supposed to be, could you refactor it out into a function?

File: runtests.py (hunk 2 above)
Comment (rossmechanic) [reply runtests.py]: {} is the default, which would lead us back to square one with the original issue

**Pull request merged**